From edceb5d51d98105d5acf71a073f7892d59eb7dfd Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 20 Nov 2023 09:28:05 +0200
Subject: [PATCH] [SW-0] gaudi3: add todo comment for gaudi3_hbw_memcpy

The gaudi3_hbw_memcpy is used by both preboot and
preboot_rtos. This patch adds comment to move this
code to common alone with jira number - no functional
change

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Ief6b4a2ee936f10bc14c8ca95c85273344460dae
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 7a76a57..239789e 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1184,6 +1184,7 @@
 
 u32 hl_get_pll_div_fix(void) { return 1; }
 
+/* TODO: below code is also used by preboot_rtos move this to common place SW-165981 */
 #define MEM_CPY_ERRATA_WR_OS_VAL	62	/* wr os should be 62 per HW errata GAUDI3_0315 */
 #define MEM_CPY_ERRATA_RD_OS_VAL	0	/* rd os should be 0 per HW errata GAUDI3_0306 */
 #define MEM_CPY_SIZE_LIMIT		(SZ_1M - SZ_16)
